stdlib/public/TensorFlow/Execution.swift

```diff
@@ -0,0 +1,74 @@
+//===-- Execution.swift ---------------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines public APIs regarding TensorFlow runtime support.
```

Comment:
does `CompilerRuntime.swift` have a corresponding message that it only contains internal details?

Reply:
Thanks for the great point! Done in https://github.com/apple/swift/pull/23032/commits/c855053745ade9eb6862a999fc88972090a33678.